packages/beacon-node/src/chain/options.ts

```diff
@@ -55,6 +55,11 @@ export type BlockProcessOpts = {
    */
   disableImportExecutionFcU?: boolean;
   emitPayloadAttributes?: boolean;
+
+  // Flags to use for broadcastValidation while publishing block
+  verifyOnly?: boolean;
```

Comment:
Can you document this option?
Eg: "Set to true to avoid storing blocks to db" or whatever